README.rst

```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
 
    http://statsmodels.github.io/dev/index.html
 
-
-This is currently a temporary solution that will be transformed into the new permanent location.
+**Note:** This is currently a temporary solution that will be transformed into the new permanent location.
 
 
 Main Features
 =============
 
-* linear regression models: Generalized least squares (including weighted least squares and
-  least squares with autoregressive errors), ordinary least squares.
-* glm: Generalized linear models with support for all of the one-parameter
+* Linear regression models:
+
+  - Ordinary least squares
+  - Generalized least squares
+  - Weighted least squares
+  - Least squares with autoregressive errors
+
+* GLM: Generalized linear models with support for all of the one-parameter
   exponential family distributions.
-* discrete: regression with discrete dependent variables, including Logit, Probit, MNLogit, Poisson, based on maximum likelihood estimators
+* Discrete model:
+
+  - Logit and Probit
+  - MNLogit - Multinomial logit
+  - Poisson regresion
+
 * rlm: Robust linear models with support for several M-estimators.
-* tsa: models for time series analysis
-  - univariate time series analysis: AR, ARIMA
-  - vector autoregressive models, VAR and structural VAR
-  - descriptive statistics and process models for time series analysis
-* nonparametric : (Univariate) kernel density estimators
-* datasets: Datasets to be distributed and used for examples and in testing.
-* stats: a wide range of statistical tests
+* Time Series Analysis: models for time series analysis
+
+  - Complete StateSpace modeling framework
+
+    - Seasonal ARIMA and ARIMAX models
+    - VARMA and VARMAX models
+    - Dynamic Factor models
+
+  - Markov switching models (MSAR), also known as Hidden Markov Models (HMM)
+  - Univariate time series analysis: AR, ARIMA
+  - Vector autoregressive models, VAR and structural VAR
+  - Descriptive statistics and process models for time series analysis
+
+* Nonparametric statistics: (Univariate) kernel density estimators
+* Datasets: Datasets used for examples and in testing
+* Statistics: a wide range of statistical tests
+
   - diagnostics and specification tests
   - goodness-of-fit and normality tests
   - functions for multiple testing
   - various additional statistical tests
-* iolib
-  - Tools for reading Stata .dta files into numpy arrays.
-  - printing table output to ascii, latex, and html
-* miscellaneous models
-* sandbox: statsmodels contains a sandbox folder with code in various stages of
-  developement and testing which is not considered "production ready".
-  This covers among others Mixed (repeated measures) Models, GARCH models, general method
-  of moments (GMM) estimators, kernel regression, various extensions to scipy.stats.distributions,
-  panel data models, generalized additive models and information theoretic measures.
-
 
-Where to get it
-===============
+* Principal Component Analysis
```

Comment:
I think before PCA

`* Imputation with MICE and regression on order statistic`